packages/design-system/src/Link/Link.styles.tsx

```diff
@@ -34,6 +48,9 @@ export const Styles = css<{ kind?: LinkKind }>`
   /* TODO: Remove this once blueprint is removed from the main repo. */
   color: var(--color) !important;
   text-decoration: var(--text-decoration) !important;
+  text-decoration-color: var(--text-decoration-color) !important;
+  text-decoration-skip-ink: all !important;
```

Comment:
huh, this is from chrome: https://theappsmith.slack.com/archives/C0293DVQACW/p1690275971242119?thread_ts=1690274622.371639&cid=C0293DVQACW

I think if all isn't supported, it's defaulting to auto which also does the job. So shall I leave it as it is so that it's explicit and whenever chrome does support it it will work?